vue-frontend/src/components/contribution/AnimatedCreation.vue

```diff
@@ -0,0 +1,181 @@
+        <div v-for="animation in animations.slice(0, 3)" :key="animation.id">
+          <div
+            class="tw-relative tw-group tw-cursor-pointer"
+            @click="openBlog(animation)"
+          >
+            <img
+              :src="animation.deskImages[0]"
+              :srcset="`${animation.deskImages[0]} 400w, ${animation.deskImages[1]} 800w`"
+              alt="animation"
+              class="tw-object-cover tw-w-full tw-h-full tw-rounded-tr-[50px] tw-rounded-bl-[50px]"
+              loading="lazy"
+            />
+            <div
+              class="tw-absolute tw-left-[50%] tw-top-[50%] tw-w-[0%] tw-h-[0%] tw-opacity-0 tw-rounded-tr-[50px] tw-rounded-bl-[50px] tw-bg-black-900 tw-transition-all tw-duration-1000 tw-ease tw--translate-x-1/2 tw--translate-y-1/2 group-hover:tw-w-[100%] group-hover:tw-h-[100%] group-hover:tw-opacity-[1]"
+            ></div>
+            <div
+              class="tw-absolute tw-top-[32%] lg:tw-top-[35%] tw-w-[100%] tw-h-full tw-text-center tw-font-inter-semibold tw-text-[1rem] lg:tw-text-[1.75rem] tw-leading-[1.5rem] lg:tw-leading-[2.625rem] tw-text-white tw-opacity-0 tw-transition-all tw-duration-1000 tw-translate-y-[-50%] group-hover:tw-translate-y-[0%] group-hover:tw-opacity-[1] group-hover:tw-transition-all group-hover:tw-duration-1000"
+            >
+              {{ animation.title }}
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="tw-flex tw-flex-col tw-gap-8 tw-w-[50%] tw-mt-[50px]">
+        <div v-for="animation in animations.slice(3, 6)" :key="animation.id">
+          <div
+            class="tw-relative tw-group tw-cursor-pointer"
+            @click="openBlog(animation)"
+          >
+            <img
+              @click.native="mixpanel.track('tap_whats_trending_blog')"
+              :src="animation.deskImages[0]"
+              :srcset="`${animation.deskImages[0]} 400w, ${animation.deskImages[1]} 800w`"
+              alt="animation"
+              class="tw-object-cover tw-w-full tw-h-full tw-rounded-tr-[50px] tw-rounded-bl-[50px]"
+              loading="lazy"
+            />
+            <div
+              class="tw-absolute tw-left-[50%] tw-top-[50%] tw-w-[0%] tw-h-[0%] tw-opacity-0 tw-rounded-tr-[50px] tw-rounded-bl-[50px] tw-bg-black-900 tw-transition-all tw-duration-1000 tw-ease tw--translate-x-1/2 tw--translate-y-1/2 group-hover:tw-w-[100%] group-hover:tw-h-[100%] group-hover:tw-opacity-[1]"
+            ></div>
+            <div
+              class="tw-absolute tw-top-[32%] lg:tw-top-[35%] tw-left-[4%] tw-w-[82%] lg:tw-w-[80%] tw-h-full tw-text-center tw-font-inter-bold tw-text-[1rem] lg:tw-text-[1.75rem] tw-leading-[1.5rem] lg:tw-leading-[2.625rem] tw-text-white tw-opacity-0 tw-transition-all tw-duration-1000 tw-translate-y-[-50%] group-hover:tw-translate-y-[0%] group-hover:tw-opacity-[1] group-hover:tw-transition-all group-hover:tw-duration-1000"
+            >
+              {{ animation.title }}
```

Comment:
Maybe you need to increase the font size of the animation title in md to lg device, because it shows very small font with the image.